rust/tests/namespace.ts

```diff
@@ -1340,13 +1340,7 @@ describe("namespace", () => {
       raindropsProgram: pids.MATCHES_ID,
     };
 
-    let failed = false;
-    try {
-      await namespaceProgram.joinNamespace(joinNsAccounts);
-    } catch (e) {
-      failed = true;
-    }
-    assert(failed);
+    assert.rejects(async () => await namespaceProgram.joinNamespace(joinNsAccounts));
```

Comment:
Why this need to be wrapped and awaited? Should be:

```suggestion
    await assert.rejects(namespaceProgram.joinNamespace(joinNsAccounts));
```

unless I'm forgetting something with regard to how rejects works.